app/src/main/java/io/intrepid/contest/screens/contestjudging/EntryListAdapter.java

```diff
@@ -1,19 +1,33 @@
 package io.intrepid.contest.screens.contestjudging;
 
+import android.support.annotation.LayoutRes;
 import android.support.v7.widget.RecyclerView;
 import android.view.ViewGroup;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import io.intrepid.contest.R;
 import io.intrepid.contest.models.Entry;
 
 public class EntryListAdapter extends RecyclerView.Adapter<EntryViewHolder> {
     private final List<Entry> entries = new ArrayList<>();
+    private final EntryOnClickListener listener;
+    @LayoutRes private final int rowLayout;
+
+    public EntryListAdapter(EntryOnClickListener listener){
+        this.listener = listener;
+        this.rowLayout = R.layout.entry_submission_row;
+    }
+
+    public EntryListAdapter(@LayoutRes int rowLayout, EntryOnClickListener listener){
```

Comment:
Same issue as with the other VH. Maybe we can sit down and talk a bit more about how to route communication between the ViewHolder and Presenter